tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -1336,6 +1316,10 @@ class CompilerTest {
     VERIFY_SUCCEEDED(pCompiler->Compile(
         pSource, name, entry.c_str(), profile.c_str(), args.data(), args.size(),
         opts.Defines.data(), opts.Defines.size(), nullptr, &pResult));
+    if (pResult == nullptr) {
+      WEX::Logging::Log::Comment(L"Failed to compile - pResult NULL");
+      return;
```

Comment:
It looks like this just logs a comment. Will it mark the test as FAILED?

I wonder if you should do the following instead?
```cpp
VERIFY_IS_TRUE(pResult!=nullptr, L"Failed to compile - pResult NULL");
```
[doc](https://docs.microsoft.com/en-us/windows-hardware/drivers/taef/verify)

Reply:
Sure.